Guide/recipes.markdown

```diff
@@ -430,7 +430,14 @@ case post of
 
 ## Highlight the targeted element
 
-Let's say you have a page with comments, and you link to them with `<a href="#comment-<comment ID>">`.
+Let's say you have a page with comments, and you link to them with `<a href="#comment-<comment ID>">`. You can build it like this (assuming you have a `comment.id`):
+
+```haskell
+[hsx|
+<a { ...[ ("href", "#comment-" ++ show comment.id)
+        , ("id", "comment-" ++ show comment.id)
+        ]}
+|]
```

Comment:
Is there any reason for the spread `...` syntax here?

```
<a href={"#comment-" ++ show comment.id} id={"comment-" ++ show comment.id}>
```

looks simpler to me

Reply:
Right, for some reason I didn't think of concatenation inside `{}`. I'll adjust